aether-kernel/aether/kernel/api/views.py

```diff
@@ -612,3 +638,38 @@ def run_mapping_validation(submission_payload, mapping_definition, schemas):
         })
     except Exception as e:
         return Response(str(e), status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@api_view(['POST'])
+@renderer_classes([JSONRenderer])
+@permission_classes([permissions.IsAuthenticated])
+def mapping_schema_unique_usage_view(request, *args, **kwargs):
```

Comment:
Why isn't this method declared inside the `SchemaViewSet` as a custom action? with that you don't need to indicate the renderer classes, the permission classes... and you take advantage of all the things we made for multitenancy, security, gateway... you could also take advantage of the filtering and search already implemented.

```python
@action(detail=True, methods=['get', 'post'], url_path='unique-usage')
def unique_usage(self, request, pk=None, *args, **kwargs):
    # return the list of schemas that are reference only once, in one project, in one mappingset or in one mapping...
```